Paperboy/Example.html

```diff
@@ -0,0 +1,14 @@
+---
+# This is an example file and gets skipped by the paperboyBuilder
+title: Title goes here 			# {string}  required - Title of the message
+author: Your name 				# {string}  optional - Name of the author
+type: information				# {string}  required - Type of message (information, extension_updates, weekly_news, developers, important)
+version: 1						# {integer} optional - Version of the message, increment if changes have been made to the file
+date: 0							# {integer} optional - Date as Unix timestamp
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1437010/12624888/9624e19e-c531-11e5-910f-a04e465bc7c7.png)
not there for me